internal/pkg/service/stream/storage/node/coordinator/fileimport/operator.go

```diff
@@ -237,10 +237,17 @@ func (o *operator) importFile(ctx context.Context, file *fileData) {
 
 	var err error
 	if !file.IsEmpty {
-		// Import the file to specific provider
-		err = o.plugins.ImportFile(ctx, &file.File)
+		// Get file statistics
+		var stats statistics.Aggregated
```

Comment:
Offtopic: in the code bellow, there is `o.locks.NewMutex(fmt.Sprintf("operator.sink.file.%s", file.FileKey.SinkKey))`, you said you wanted to do it in a centralized way, so the name of the lock will be defined only in one place.